Dockerfile

```diff
@@ -1,5 +1,12 @@
+FROM maven:3-eclipse-temurin-17 AS builder
```

Comment:
```suggestion
FROM maven:3-eclipse-temurin-17-alpine AS builder
```

I expect the image to be slightly smaller, but Java is the big fat thing here. anyway...